package.json

```diff
@@ -32,24 +32,26 @@
     "hps": "local-ssl-proxy --cert ./dev/localhost.pem --key ./dev/localhost-key.pem"
   },
   "dependencies": {
+    "decanter": "^7.0.0-rc.3",
```

Comment:
Do we have anything coming out of the decanter package that is needed during runtime? This might actually be a devDependency.

Reply:
I'll test this out as well 👍🏼 😄